### PR TITLE
Add basic structure

### PR DESCRIPTION
**Pull request by christophschmitz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,67 @@
...
+            </ul>
+        </section>
+        <section>
+            <h2>Wanderregionen Schwarzwald</h2>
+            <p>Der Schwarzwald ist Deutschlands höchstes und größtes zusammenhängendes Mittelgebirge und liegt im
+                Südwesten Baden-Württembergs.</p>
+            <h3>Höchste Gipfel</h3>
+            <ul>
+                <li>Kandl</li>
+                <li>Schau ins land</li>
```
Comment (JuliaJau) [inline index.html:50]:
```suggestion
                <li>Schau ins land</li>
```Schauinsland

File: index.html [hunk 2]
```diff
@@ -1,12 +1,67 @@
...
+                <li>Belchen</li>
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <h4>Quellen</h4>
+        <nav>
+            <ul>
+                <li> <a href="https://de.wikipedia.org/wiki/Alpen">Alpen</a></li>
```
Comment (JuliaJau) [inline index.html:59]:
```suggestion
                <li> <a href="https://de.wikipedia.org/wiki/Alpen">Alpen</a></li>
```<li><a href="https://de.wikipedia.org/wiki/Alpen">Alpen</a></li>

File: index.html [hunk 3]
```diff
@@ -1,12 +1,67 @@
...
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wanderrouten</title>
+    <title>Wanderregionen in Deutschland</title>
 </head>
+<header>
+    <nav>
+        <ul>
+            <li> <a href="#">Alpen</a></li>
```
Comment (JuliaJau) [inline index.html:13]:
```suggestion
            <li> <a href="#">Alpen</a></li>
```<li><a href="#">Alpen</a></li>

Comment (christophschmitz) [reply index.html:13]: Thank you! You are right!

**Pull request merged**